src/main/webapp/js/instructorFeedbackResults.es6

```diff
@@ -1,40 +1,50 @@
-'use strict';
-
-$(document).ready(function() {
-    var participantPanelType = 'div.panel.panel-primary,div.panel.panel-default';
-
-    $('a[id^="collapse-panels-button-section-"]').on('click', function() {
-        var isGroupByTeam = document.getElementById('frgroupbyteam').checked;
-        var childPanelType;
-        if (isGroupByTeam) {
-            childPanelType = 'div.panel.panel-warning';
-        } else {
-            childPanelType = participantPanelType;
-        }
-        var panels = $(this).closest('.panel-success')
+/* global selectElementContents:false,
+          bindCollapseEvents:false,
+          bindPublishButtons:false,
+          bindUnpublishButtons:false,
+          setStatusMessage:false,
+          showSingleCollapse:false,
+          hideSingleCollapse:false,
+          toggleSingleCollapse:false
+*/
+/* eslint-disable no-use-before-define */
+/* exported     submitFormAjax,
+                updateStatsCheckBox,
+                getNextId,
+                displayAjaxRetryMessageForPanelHeading,
+                isEmptySection,
+                removeSection
+*/
+
+$(document).ready(() => {
+    const participantPanelType = 'div.panel.panel-primary,div.panel.panel-default';
+
+    $('a[id^="collapse-panels-button-section-"]').on('click', (e) => {
+        const isGroupByTeam = document.getElementById('frgroupbyteam').checked;
+        const childPanelType = isGroupByTeam ? 'div.panel.panel-warning' : participantPanelType;
+        const panels = $(e.currentTarget).closest('.panel-success')
                             .children('.panel-collapse')
                             .find(childPanelType)
                             .children('.panel-collapse');
-        toggleCollapse(this, panels);
+        toggleCollapse(e.currentTarget, panels);
     });
 
-    $('.panel.panel-success').on('click', 'a[id^="collapse-panels-button-team-"]', function() {
-        var panels = $(this).closest('.panel-warning')
+    $('.panel.panel-success').on('click', 'a[id^="collapse-panels-button-team-"]', (e) => {
+        const panels = $(e.currentTarget).closest('.panel-warning')
                             .children('.panel-collapse')
                             .find(participantPanelType)
                             .children('.panel-collapse');
-        toggleCollapse(this, panels);
+        toggleCollapse(e.currentTarget, panels);
     });
 
-    $('#results-search-box').keyup(function() {
-        updateResultsFilter();
-    });
+    $('#results-search-box').keyup(updateResultsFilter);
 
     // prevent submitting form when enter is pressed.
-    $('#results-search-box').keypress(function(e) {
+    $('#results-search-box').keypress((e) => {
         if (e.which === 13) {
             return false;
```

Comment:
Why? In this situation there is no difference. Or is it because it's an unnecessary change (which it technically is xD).